src/AppBundle/DataFixtures/ORM/PostFixtures.php

```diff
@@ -47,6 +47,7 @@ public function load(ObjectManager $manager)
 
             $post->setTitle($title);
             $post->setSummary($this->getRandomPostSummary());
+            // TODO: fix this 'slugger' call
```

Comment:
Autowiring works for services. Fixture classes have nothing to do with the DI container